README.rst

```diff
@@ -117,15 +117,15 @@ not required for Mac builds.
 
 Build the dependencies::
 
-    $ OSX_ONLY=1 $O3/core/scripts/mac/build-all
+    $ OSX_ONLY=1 DL=~/src/mac $O3/core/scripts/mac/build-all
```

Comment:
ah this is the source of the problem. By doing so, you are using a DL folder equal to the default value of  DEP_DIR.
DL and DEP_DIR must be two different directories. Maybe this should be clarified in the README.